client/landing/gutenboarding/stores/onboard/resolvers.ts

```diff
@@ -21,7 +23,24 @@ import { receiveVerticals } from './actions';
  */
 export function* getVerticals() {
 	const url = 'https://public-api.wordpress.com/wpcom/v2/verticals';
-	const verticals = yield apiFetch( { url } );
+	let locale;
+	if ( ! locale && typeof navigator === 'object' && 'languages' in navigator ) {
+		for ( const langSlug of navigator.languages ) {
+			const language = getLanguage( langSlug.toLowerCase() );
+			if ( language ) {
+				locale = language.langSlug;
+				break;
+			}
+		}
+	}
+
+	let localeQueryParam;
+	if ( locale && locale !== 'en' ) {
+		// v2 api request
+		localeQueryParam = { _locale: locale };
+	}
+
+	const verticals = yield apiFetch( { url: addQueryArgs( url, localeQueryParam ) } );
```

Comment:
We'll probably want to implement this as `apiFetch` middleware. Logged in users would have no change (`_locale=user` is already included), but logged out users would try to find an appropriate locale.